standard/sections/clause_7_normative_text.adoc

```diff
@@ -739,18 +747,79 @@ include::../recommendations/core/REC_data_policy.adoc[]
 }]
 ----
 
+include::../requirements/core/REQ_data_policy.adoc[]
+include::../recommendations/core/PER_data_policy.adoc[]
+
 ==== Links and Distribution Information [[links-distribution]]
 
 ===== Overview
 
-The `+links+` property describes URLs and APIs for accessing or subscribing to the dataset.
+The `+links+` property describes URLs and APIs for accessing or subscribing to the dataset.  Links are a core
+mechanism for further interacting with data, and provide "actionable" workflow to enhance and improve the
+user experience with "less clicks" to subscribe to or download data.
```

Comment:
add a sentence describing other options. e.g. 

"The links section property may also describe URLs for related documentation, visual images of the dataset or related data."

Reply:
Fixed/updated.